services/src/datasets/external/netcdfcf/overviews.rs

```diff
@@ -208,93 +215,105 @@ impl NetCdfVisitor for Group<'_> {
     }
 }
 
-pub fn create_overviews<C: TaskContext>(
+pub async fn create_overviews<C: TaskContext + 'static>(
     provider_path: &Path,
     dataset_path: &Path,
     overview_path: &Path,
     resampling_method: Option<ResamplingMethod>,
-    task_context: &C,
+    task_context: C,
+    db_transaction: &Transaction<'_>,
 ) -> Result<OverviewGeneration> {
+    if overview_exists(dataset_path, db_transaction).await? {
+        debug!("Skipping conversion: {}", dataset_path.display());
```

Comment:
add the reason maybe

Reply:
[c40d046](https://github.com/geo-engine/geoengine/pull/931/commits/c40d0460bdfe297bd67dc8328d91766f294d8736)

its gone now :shrug: